graylog2-web-interface/src/components/configurationforms/ConfigurationForm.jsx

```diff
@@ -100,6 +103,22 @@ class ConfigurationForm extends React.Component {
     return diff;
   };
 
+  _renderConfigFields = (configFields, autoFocus) => {
+    let shouldAutoFocus = autoFocus;
+    const configFieldKeys = $.map(configFields, (field, name) => name)
```

Comment:
Let's use JS [`Array.map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) instead of jQuery